### PR TITLE
Course Outline Empty State

### PR DESCRIPTION
**Pull request by ShaharTal**

### CODE REVIEW

File: src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineListEmptyState/index.tsx [hunk 1]
```diff
@@ -24,13 +21,13 @@ export default function CourseOutlineListEmptyState({
...
   return (
-    <WMEmpty
-      description="Start building your course by creating lessons and dragging items from the Items List"
-      image={<Icon type={IconType.EmptyCourse} />}
-      className={className}
-      containerClassName={containerClassName}
-    >
-      <ActionMenu onActionSelected={onActionSelected} />
-    </WMEmpty>
+    <div className={classes['course-outline-empty-state']}>
```
Comment (dvira) [inline src/components/Screen-v2/CourseEditorScreen/CourseOutline/CourseOutlineTab/CourseOutlineListEmptyState/index.tsx:24]: Please write classNames in camelCase.

File: src/components/common-v2/buttons/CircleButton/index.tsx [hunk 2]
```diff
@@ -4,41 +4,44 @@ import {
...
   WMTooltip,
+  WMTooltipProps,
 } from '@walkme/wm-ui';
 
 import { useAppSkeleton } from '../../../../hooks/skeleton';
 
 import classes from './style.module.scss';
 
-function ExportButton({
+export interface ICircleButtonProps extends WMButtonProps {
```
Comment (dvira) [inline src/components/common-v2/buttons/CircleButton/index.tsx:16]: We no longer prefix interfaces with 'I', please remove it.

**Review by dvira — APPROVED** [review]
Looks good, approved with some minor comments.

File: src/components/common-v2/buttons/CircleButton/index.tsx (hunk 2 above)
Comment (ShaharTal) [reply src/components/common-v2/buttons/CircleButton/index.tsx:16]: Nobody told me! :) Will do

**Review by dvira — APPROVED** [review]
Great!

**Pull request merged**